docs/pages/components/AnchoredOverlay.mdx

```diff
@@ -188,6 +188,11 @@ function AnchoredOverlayWithOffsetExample() {
       type: 'object',
       description: 'When `focusTrap` is true, optional arguments to configure the focus trap.'
     },
+    {
+      name: 'portal',
+      type: ['React.Ref', 'HTMLElement'],
+      description: 'The parent element to place the AnchoredOverlay in.'
+    },
```

Comment:
Does this not need to include the type returned from [`createPortal`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/91176960584b69402f0e7c8745436b5d59caeebd/types/react/v18/index.d.ts#L378-L380) in this case a `ReactElement`?